src/main/java/mods/railcraft/common/blocks/machine/alpha/TileSteamTrap.java

```diff
@@ -76,6 +78,15 @@ public void updateEntity() {
         }
         triggerCheck();
         if (isJetting()) for (EntityLivingBase entity : getEntitiesInSteamArea()) {
+            if (Loader.isModLoaded("gregtech")) {
```

Comment:
I suggest restructuring this slightly to avoid extra checks, as well as adding a GT6 check:
```java
if (isJetting()) {
    boolean gt5Loaded = Loader.isModLoaded("gregtech") && !Loader.isModLoaded("gregapi");
    for (EntityLivingBase entity : getEntitiesInSteamArea()) {
        if (gt5Loaded) {
            // ...
        }
        entity.attackEntityFrom(RailcraftDamageSource.STEAM, DAMAGE);
    }
}
```